jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClient.java

```diff
@@ -474,9 +475,11 @@ public IQ handleIQRequest(IQ iqRequest)
      * Handles an IQ received from Smack by passing it to the listener which is
      * registered.
      * @param iq the IQ to handle.
+     * @param requireResponse whether to send an error stanza as a response if the {@link IQListener} produces
+     * {@code null}.
      * @return the response.
      */
-    private IQ handleIq(IQ iq)
+    private IQ handleIq(IQ iq, boolean requireResponse)
```

Comment:
Do we want to add an overload that doesn't have the `requireResponse` parameter, to keep from having to update callers in places which don't need it?  (Specifically it seems like jicofo will be a pain in the PR tests otherwise.)

Reply:
Theree are overloads for the public methods (`MucClientManager#registerIQ`), this one is private.